cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -23,3 +31,45 @@ spec:
         resources:
           requests:
             storage: 10Gi
+  cephVersion:
+    image: ceph/ceph:v14.2.1-20190430
+    allowUnsupported: false
+  dashboard:
+    enabled: true
+  network:
+    hostNetwork: false
+  storage:
+    storageClassDeviceSets:
+    - name: cluster1-set1
+      count: 3
+      resources:
+      #   limits:
+      #     cpu: "500m"
+      #     memory: "4Gi"
+      #   requests:
+      #     cpu: "500m"
+      #     memory: "4Gi"
+      # placement:
+      #   podAntiAffinity:
+      #     preferredDuringSchedulingIgnoredDuringExecution:
+      #     - weight: 100
+      #       podAffinityTerm:
+      #         labelSelector:
+      #           matchExpressions:
+      #           - key: "rook.io/cluster"
+      #             operator: In
+      #             values:
+      #               - cluster1
+      #           topologyKey: "failure-domain.beta.kubernetes.io/zone"
+      volumeClaimTemplates:
+      - metadata:
+          name: template-1
```

Comment:
Let's name the example template "data". It really is for the data backing the OSD. In the future we would likely want to support "data" and "metadata" anyway.

Reply:
updated.